Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM php:7.0-apache
+
+MAINTAINER Omar Padron "omar.padron@kitware.com"
```

Comment:
@opadron Something more generalized like, Kitware "software@kitware.com" is probably more appropriate here but looping in @bradking to enquire about the correct email to use. Also https://docs.docker.com/engine/reference/builder/#maintainer-deprecated.

Reply:
I've updated the maintainer section.  I'll adjust further if needed.